pkg/targets/reconciler/tektontarget/reaper.go

```diff
@@ -40,27 +40,27 @@ func reaperThread(ctx context.Context, r *reconciler) {
 
 	p, err := cloudevents.NewHTTP()
 	if err != nil {
-		log.Fatal("failed to create new HTTP based cloudevent: ", zap.Error(err))
+		log.Fatalw("Failed to create new HTTP based Cloudevent: ", zap.Error(err))
 	}
 
 	client, err := cloudevents.NewClient(p)
 	if err != nil {
-		log.Fatal("Unable to create cloudevent client: ", zap.Error(err))
+		log.Fatalw("Unable to create Cloudevent client: ", zap.Error(err))
 	}
 
 	for {
 		<-poll.C // Used to wait for the poll timer
 		log.Debug("executing reaping")
 		nsl, err := k8sclient.Get(ctx).CoreV1().Namespaces().List(ctx, v1.ListOptions{})
 		if err != nil {
-			log.Fatal("Unable to retrieve namespaces: ", zap.Error(err))
+			log.Fatalw("Unable to retrieve namespaces: ", zap.Error(err))
 		}
 
 		// search for tektontargets across all namespaces
 		for _, ns := range nsl.Items {
 			targets, err := r.targetLister(ns.Name).List(labels.Everything())
 			if err != nil {
-				log.Warn("unable to retrieve targets for #{ns.Name}: ", zap.Error(err))
+				log.Warnw("unable to retrieve targets for #{ns.Name}: ", zap.Error(err))
```

Comment:
Not your code, but I am puzzled about this. What does "... #{ns.Name}" do?
Either there is something at zap that I am not aware of or this can be replaced with


```suggestion
				log.Warnw("unable to retrieve target adapters", zap.Error(err), zap.String("namespace",ns.Name))
```

Reply:
The syntax inside the log string looks like a ruby in-line variable substitution that has long since been deprecated/removed. Your suggestion makes sense.